local-cli/wrong-react-native.js

```diff
@@ -9,8 +9,21 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-var script = process.argv[1];
-var installedGlobally = script.indexOf('node_modules/.bin/react-native') === -1;
+const fs = require('fs');
+const path = require('path');
```

Comment:
you can move these requires inside of the windows code, so they won't be required on non-win32.

Not a huge refactor, but I'd do it.

Reply:
Agreed, and done.